legal/refund-policy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Refund Policy
+layout: en
+no_toc: true
```

Comment:
This means the page doesn't get the in page table of contents, or list of H2s - like the other pages in legal. I'm not really sure why we do that though.